src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts = [] }) => (
+
+  <ul className="PostList">
+    {posts.map(post => (
+      <li
+        key={post.id}
+        className="PostInfo"
+      >
+        <PostInfo
+          title={post.title}
+          body={post.body}
+          user={post.user}
+          comments={post.comments}
```

Comment:
use destructuring

Reply:
Hi, Vlad! Thank you for the review and recommendations! I have implemented them.